src/org/parosproxy/paros/view/View.java

```diff
@@ -1042,4 +1047,13 @@ public void displayMessage(Message message) {
             getResponsePanel().setMessage(httpMessage, true);
         }
     }
+
+
+    public boolean isCanGetFocus() {

```

Comment:
It could have a since JavaDoc tag (and a little description, like the var has), same for following method.